modules/ObjectControl/src/objectcontrol.cpp

```diff
@@ -169,6 +172,20 @@ void ObjectControl::onStopMessage(const Stop::message_type::SharedPtr){
 	this->tryHandleMessage(f_try,f_catch, Stop::topicName, get_logger());	
 }
 
+void ObjectControl::onResetTestMessage(const ResetTest::message_type::SharedPtr) {
+	COMMAND cmd = COMM_BACKTOSTART_CALL;
```

Comment:
I noticed that it is the same command here and at `onReloadSettingsMessage`, is this correct?

Reply:
They are both related to the reset functionality but I changed it to `COMM_OSEM`, should hopefully be more accurate?